gapii/cc/gles_mid_execution.cpp

```diff
@@ -224,10 +204,54 @@ class SamplerCube : public Sampler {
   GLint mFace;
 };
 
+typedef struct {
```

Comment:
The use of `typedef struct` is only really used in C code. Just use `struct texture_state { ... }`